R/plot-cumulative-time-profile.R

```diff
@@ -99,15 +99,25 @@ plotCumulativeTimeProfile <- function(data = NULL,
 
   # And optional color palette otherwise use colors from theme
   if (!isEmpty(plotConfiguration$colorPalette)) {
-    try(suppressMessages(
-      plotObject <- plotObject +
-        ggplot2::scale_fill_brewer(
-          palette = plotConfiguration$colorPalette,
-          aesthetics = c("color", "fill")
-        )
-    ))
+    if(!isIncluded(plotConfiguration$colorPalette, .ViridisPalettes)){
+      try(suppressMessages(
```

Comment:
What happens when `try` fails?

Reply:
You get only a warning message:

```
Warning message:
In pal_name(palette, type) : Unknown palette xxx
```

Reply:
`suppressMessages` is only intended to prevent the ggplot annoying message stating that there was already a scale on colors

Reply:
But then no palette is set? I am just confused that there is no `catch` for the `try`.

Reply:
Indeed, then I have set a `tryCatch` instead. Here is corresponding outcome from the simple example below:
```r
plotTornado(
    x = rnorm(5),
    y = letters[1:5],
    colorPalette = "xxx"
)
```
```
Warning message:
Unknown palette 'xxx', using palette 'Set1' instead.
See enum 'ColorPalettes' to check available color palettes.
```
![image](https://user-images.githubusercontent.com/45343665/231973154-649c2851-59ff-49d4-92dd-f2b5a5b21843.png)